frontend/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "sourceMap": true,
     "strict": true,
     "preserveSymlinks": true,
-    "types": ["vitest/globals"],
-    "moduleResolution": "Node"
```

Comment:
Svelte recommends to stick with the more future-proof 'Bundler' resolution mode that it has defined by default.